src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import core.basesyntax.dataservices.DataConverter;
+import core.basesyntax.dataservices.DataConverterImpl;
+import core.basesyntax.dataservices.DataProcess;
+import core.basesyntax.dataservices.DataProcessImpl;
+import core.basesyntax.dataservices.DataReader;
+import core.basesyntax.dataservices.DataReaderImpl;
+import core.basesyntax.reportservices.ReportCreator;
+import core.basesyntax.reportservices.ReportCreatorImpl;
+import core.basesyntax.reportservices.ReportToFile;
+import core.basesyntax.reportservices.ReportToFileImpl;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        String inputFilePath = "src/main/resources/input.csv";
+        String outputFilePath = "src/main/resources/report.csv";
+
+        DataReader dataReader = new DataReaderImpl();
+        DataConverter dataConverter = new DataConverterImpl();
+        DataProcess dataProcess = new DataProcessImpl();
+        ReportCreator reportCreator = new ReportCreatorImpl();
+        ReportToFile reportToFile = new ReportToFileImpl();
+
+        List<String> readFile = dataReader.read(inputFilePath);
+        List<FruitTransaction> transactions = dataConverter.convert(readFile);
+        Map<String,Integer> processedTransactions = dataProcess.process(transactions);
+        List<String> report = reportCreator.report(processedTransactions);
+        reportToFile.writeReportToFile(report, outputFilePath);
+    }
```

Comment:
The `Main` class should not instantiate all services directly in the `main()` method. Instead, consider using a dependency injection mechanism to make the code more modular and testable. You could create a configuration class or use a factory pattern to instantiate these services.